### PR TITLE
Added network and filesystem to utilization graphs

### PR DESCRIPTION
**Pull request by cloudbehl**
Screenshot from 2019-10-23 14-42-51

Signed-off-by: cloudbehl <cloudbehl@gmail.com>

### CODE REVIEW

File: frontend/public/components/dashboard/project-dashboard/utilization-card.tsx [hunk 1]
```diff
@@ -41,16 +47,46 @@ export const UtilizationCard = withDashboardResources(
...
+      queries[ProjectQueries.FILESYSTEM_USAGE],
+      'loadError',
+    ]);
     const podCount = prometheusResults.getIn([
       queries[ProjectQueries.POD_COUNT],
       'data',
     ]) as PrometheusResponse;
     const podCountError = prometheusResults.getIn([queries[ProjectQueries.POD_COUNT], 'loadError']);
 
+    const networkCustomQuery =
```
Comment (rawagner) [inline frontend/public/components/dashboard/project-dashboard/utilization-card.tsx]: move this to `queries.ts`

File: frontend/public/components/graphs/area.tsx [hunk 2]
```diff
@@ -121,6 +127,7 @@ type AreaChartProps = {
   tickCount?: number;
   title?: string;
   data?: DataPoint[];
```
Comment (rawagner) [inline frontend/public/components/graphs/area.tsx]: how about changing this to
```suggestion
  data?: DataPoint[][];
```

and then having
```javascript
<ChartGroup>
  {data.map((datum) => (
    <ChartArea data={datum} style={style} />
   ))}
</ChartGroup>
```
in render

Comment (rawagner) [reply frontend/public/components/graphs/area.tsx]: I would also use the same approach in UtilizationItem - `data` would be something like

```
data: {
  type: string //In, Out..
  data: DataPoint[]
}[]
```

File: frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx [hunk 3]
```diff
@@ -43,11 +69,12 @@ export const UtilizationItem: React.FC<UtilizationItemProps> = React.memo(
         <div className="co-utilization-card__item__section">
           <div className="pf-l-level">
             <h4 className="pf-c-title pf-m-md">{title}</h4>
-            {current}
+            {datacurrent}
           </div>
           <div className="pf-l-level">
             <span className="co-utilization-card__item__text" />
             <span className="co-utilization-card__item__text">
+              {data1current && <span>{data1current}</span>}
```
Comment (rawagner) [inline frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx]: `data1current` should not be grayed out

**Comment by mareklibra** [comment]
Please note, I am adding multi-line charts within #3089

**Comment by cloudbehl** [comment]
@mareklibra then will wait for it to get merged
@rawagner ^^

**Comment by cloudbehl** [comment]
closing re: https://github.com/openshift/console/pull/3210

**Pull request closed without merging**